### PR TITLE
Station map

### PR DESCRIPTION
**Pull request by RobinLinde**
Adds a new (hidden) theme showing train stations and their indoor mapping. Also allows disabling presets when re-using a layer.

Features:
- Ticket machines
- Ticket validators
- ATMs
- Car Rental
- Platforms
- Departure Boards

Some well mapped locations:
- [Berlin HBF](https://mapcomplete-git-feature-station-map-robinlinde.vercel.app/stations?z=20&lat=52.52454359620789&lon=13.369504362344744)
- [Amsterdam C.](https://mapcomplete-git-feature-station-map-robinlinde.vercel.app/stations.html?z=20&lat=52.3785&lon=4.900261)
- [Assen](https://mapcomplete-git-feature-station-map-robinlinde.vercel.app/stations.html?z=21&lat=52.99237&lon=6.570929)
- [Grijpskerk](https://mapcomplete-git-feature-station-map-robinlinde.vercel.app/stations.html?z=20&lat=53.2559&lon=6.310312) (without any indoor mapping)

### CODE REVIEW

**Comment by pietervdvn** [comment]
Wow, what an amazing piece of work!

Is there a reason why you marked this as "Work in Progress"?

**Comment by RobinLinde** [comment]
There are still some details I'd like to fix, e.g. on most ticket machines here payment is possible using coins, but not using notes, so I'd like to change/create a new tagrendering for that, maybe one for ` payment:coins:denominations`/`payment:notes:denominations` as well.

**Comment by RobinLinde** [comment]
Most of the stuff I mentioned is done now, so should be finished (for now)

File: assets/layers/ticket_machine/ticket_machine.json [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+          },
+          "addExtraTags": [
+            "operator:wikidata=Q23076"
+          ],
+          "hideInAnswer": "_country!=nl"
+        }
+      ]
+    },
+    {
+      "id": "payment-options",
```
Comment (pietervdvn) [inline assets/layers/ticket_machine/ticket_machine.json:56]: A 'payment-options' already exists with the reusable 'questions'-json file; maybe this one is more appropriate there?

File: assets/layers/ticket_machine/ticket_machine.json [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+          "icon": "./assets/tagRenderings/payment_card.svg",
+          "then": {
+            "en": "You can pay here using cards"
+          },
+          "hideInAnswer": true
+        }
+      ]
+    },
+    {
+      "id": "denominations-coins",
```
Comment (pietervdvn) [inline assets/layers/ticket_machine/ticket_machine.json:115]: This one would be well-suited for the shared questions.json as well

**Review by pietervdvn — CHANGES_REQUESTED** [review]
Really outstanding work - once again!

I'm afraid that asking a 'name' for an ATM will result in many operators/brands being filled out; so maybe we should drop this question.

Furthermore, moving a few question to the 'tagRenderings/questions.json' file for reuse would be nice

File: assets/layers/ticket_machine/ticket_machine.json (hunk 2 above)
Comment (RobinLinde) [reply assets/layers/ticket_machine/ticket_machine.json]: Moved this one over as well

File: assets/layers/ticket_machine/ticket_machine.json (hunk 1 above)
Comment (RobinLinde) [reply assets/layers/ticket_machine/ticket_machine.json]: I think for most items asking `payment:cash` is enough but some automated stuff might only have coin slots (or the other way around). I've moved it to questions.json, reusing the question and the original mappings (but hiding them)

File: assets/layers/atm/atm.json [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        "amenity=atm"
+      ],
+      "title": {
+        "en": "an ATM"
+      }
+    }
+  ],
+  "tagRenderings": [
+    {
+      "id": "name",
```
[the remark this answers is not in the document]
Comment (pietervdvn) [reply assets/layers/atm/atm.json:38]: That is an excellent compromise.

**Review by pietervdvn — APPROVED** [review]

**Pull request merged**